### ISSUE TITLE
About the Pre-trained caffemodel only on ActivityNet_v1.2 training set.

### ISSUE THREAD

**Issue by zaie** [wanglimin/UntrimmedNet#28]
Hello，I have the same question of #16 . The link you provided didn't work anymore. Can you please provide another link for the Pre-trained caffemodel only on ActivityNet_v1.2 training set? Thanks.

**Comment by wanglimin** [comment]
The models could be downloaded from:

http://mcg.nju.edu.cn/models/UntrimmedNet/anet1.2_spatial_untrimmednet_hard_bn_inception.caffemodel
http://mcg.nju.edu.cn/models/UntrimmedNet/anet1.2_spatial_untrimmednet_soft_bn_inception.caffemodel
http://mcg.nju.edu.cn/models/UntrimmedNet/anet1.2_temporal_untrimmednet_hard_bn_inception.caffemodel
http://mcg.nju.edu.cn/models/UntrimmedNet/anet1.2_temporal_untrimmednet_soft_bn_inception.caffemodel

**Comment by zaie** [comment]
Thanks. But I got "You don't have permission to access /models/UntrimmedNet/anet1.2_temporal_untrimmednet_hard_bn_inception.caffemodel on this server."

**Comment by wanglimin** [comment]
If this link does not work, please download them from dropbox:

https://www.dropbox.com/sh/8yxjj4fbgm2omjd/AADA_ebg-pciLSoC6dUoT1cwa?dl=0

**Comment by zaie** [comment]
It works! Thanks a lot!

**Comment by AmoseKang** [comment]
Thank you for your update. But it seems your anet models in dropbox are as same as the former one(training on both training and validation set), they have same md5sum and so far reached nearly 100% accuracy in activity net validation set.

**Comment by wanglimin** [comment]
I will look for the model trained solely on the train set. We suggest to report performance on the test set as its groudtruth is not public.

**Comment by wanglimin** [comment]
The models on ActivityNet trained only on train set are at:
http://mcg.nju.edu.cn/models/UntrimmedNet/anet1.2_train_rgb_weak_tsn_bn_inception_average_seg3_attention_sw7_iter_10000.caffemodel
http://mcg.nju.edu.cn/models/UntrimmedNet/anet1.2_train_rgb_weak_tsn_bn_inception_average_seg3_top3_sw7_iter_10000.caffemodel
http://mcg.nju.edu.cn/models/UntrimmedNet/anet1.2_train_flow_weak_tsn_bn_inception_average_seg3_attention_sw7_iter_18000.caffemodel
http://mcg.nju.edu.cn/models/UntrimmedNet/anet1.2_train_flow_weak_tsn_bn_inception_average_seg3_top3_sw7_iter_18000.caffemodel
